### PR TITLE
Classes - Answer and Test Files

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

File: answers/Classes/CorrectMultipleErrorsTest.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
```
Comment (TenshiLyle) [inline answers/Classes/CorrectMultipleErrorsTest.php:10]: what is the use of the `myMethod` in here?

File: answers/Classes/CreateSimpleClassTest.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+    class Person 
+    {
+        public $name = "Diana";
```
Comment (TenshiLyle) [inline answers/Classes/CreateSimpleClassTest.php:4]: What is the use of `$name` property in here?

File: answers/Classes/MissingAssignmentOperatorTest.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+    class MyClass 
+    {
+        public $myProp = "This is a class property.";
+        
+        public function myMethod() 
+        {
+            echo "This is a class method.";
+        }
+    }
```
Comment (TenshiLyle) [inline answers/Classes/MissingAssignmentOperatorTest.php:10]: Same as this one, what is the use of the `myMethod` in here?

File: answers/Classes/CorrectMultipleErrorsTest.php (hunk 1 above)
Comment (cerilacanda) [reply answers/Classes/CorrectMultipleErrorsTest.php:10]: This is the name of the method in the sample class created. Please check this one. Thank you.

image

File: answers/Classes/CreateSimpleClassTest.php (hunk 2 above)
Comment (cerilacanda) [reply answers/Classes/CreateSimpleClassTest.php:4]: This is also the sample property defined in the `Person` class as part of the sample program. Thank you.

**Review by TenshiLyle — APPROVED** [review]

**Review by TenshiLyle — APPROVED** [review]

**Pull request merged**